helm_deploy/prisoner-content-hub-backend/values.production.yaml

```diff
@@ -5,6 +5,11 @@ ingress:
     - host: manage.content-hub.prisoner.service.justice.gov.uk
       cert_secret: prisoner-content-hub-cms-certificate
     - host: cms-prisoner-content-hub-production.apps.live-1.cloud-platform.service.justice.gov.uk
+  jsonapi:
+    annotations:
+      external-dns.alpha.kubernetes.io/set-identifier: prisoner-content-hub-backend-jsonapi-prisoner-content-hub-production-blue
+    hosts:
+      - host: jsonapi-cms-prisoner-content-hub-production.apps.live-1.cloud-platform.service.justice.gov.uk
```

Comment:
Do we need a host for production?  Could remove this potentially, but might be nice to be able to debug issues.